bstats-bukkit-lite/src/main/java/org/bstats/bukkit/MetricsLite.java

```diff
@@ -158,7 +165,7 @@ public void run() {
                     }
```

Comment:
If Java 7 support is removed, might as well use lambdas here too.

Reply:
hmm, missed that